clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10406,6 +10406,42 @@ void OffloadDeps::ConstructJobMultipleOutputs(Compilation &C,
   constructJob(C, JA, Outputs, Inputs, TCArgs, LinkingOutput);
 }
 
+// Utility function to gather all options for LLVM to SPIR-V translation using
+// SPIR-V backend.
+static void getSPIRVBackendOpts(const llvm::opt::ArgList &TCArgs,
+                                ArgStringList &TranslatorArgs) {
+  TranslatorArgs.push_back(TCArgs.MakeArgString("-filetype=obj"));
+  TranslatorArgs.push_back(
+      TCArgs.MakeArgString("-mtriple=spirv64-unknown-unknown"));
+  TranslatorArgs.push_back(TCArgs.MakeArgString("-O0"));
```

Comment:
Should we add a comment we are forcing O0 due to higher opt levels hitting issues in the backend (I assume that's why this was added)?

Reply:
I expect this section to be updated often as we gain more confidence in testing and also keep adding feature supports. I will add a generic comment for this. Thanks